src/svgcontext.js

```diff
@@ -334,7 +336,7 @@ export class SVGContext {
       height *= -1;
     }
 
-    this.rect(x, y, width - 0.5, height - 0.5, this.attributes);
```

Comment:
This is still the case, but that is by design. Also shifting by 0.5 does not solve the problem, it just pushes the anti-aliasing problem onto lines with even-numbered widths.

In both canvas and svg, a line's origin is in the middle, ie: the width is evenly distributed on both sides. The combination of integer coordinate values with an odd pixel width ends up distributing a half-pixel on each side, causing the anti-aliasing engine takes over. So for odd numbered pixel widths, you need to shift by 0.5 to avoid anti aliasing artifacts, but for even numbered pixel widths, you don't.